framework/test/jest/unit/specs/controller/helpers/validator/validator.spec.js

```diff
@@ -31,9 +22,32 @@ const {
 const { SchemaValidationError } = require('../../../../../../../src/errors');
 
 jest.mock('ajv');
-jest.mock('ajv-keywords');
-
-describe('validator.js', () => {
+/**
+ * After completing the issue #4026, this test suite started to fail.
+ * After investigating further, I realized this particular test suite
+ * does not give meaningful feedback. Since, it's just a snapshot of the implementation.
+ *
+ * Also, we plan to remove the validator module and use "lisk-validator" instead.
+ * That's why refactoring this test would be a redundant effort at the moment.
+ * We will tackle this issue again with: https://github.com/LiskHQ/lisk-sdk/issues/4610
+ * @todo remove this test suite after introducing "lisk-validator"
```

Comment:
Please create the issue if not there already and mention in the todo comment for using `lisk-validator`

Reply:
Issue is there. See the line above: https://github.com/LiskHQ/lisk-sdk/issues/4610